x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart_with_tooltip.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState, useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { MlTooltipComponent } from '../../../components/chart_tooltip';
+import { TimeseriesChart } from './timeseries_chart';
+import { CombinedJob } from '../../../../../common/types/anomaly_detection_jobs';
+import { ml } from '../../../services/ml_api_service';
```

Comment:
As you created a functional component, could you please replace a static import with mlApiServices from the Kibana context?

Reply:
Updated here https://github.com/elastic/kibana/pull/81490/commits/3fb8c08929d8209b880f27b05db556e6a21c195b